example/package.json

```diff
@@ -7,18 +7,19 @@
   "description": "webpack-config example app",
   "main": "webpack.config.js",
   "dependencies": {
+    "extract-text-webpack-plugin": "^2.0.0-rc.3",
+    "html-webpack-plugin": "^2.28.0",
     "react": "^15.4.1",
-    "react-dom": "^15.4.1"
+    "react-dom": "^15.4.1",
+    "write-file-webpack-plugin": "^3.4.2"
```

Comment:
No need to use `write-file-webpack-plugin`. Webpack dev server keeps files in memory, without touching the file system. This is faster then writing to FS on every re-build.